stdpopsim/catalog/HomSap/dfes.py

```diff
@@ -147,3 +147,59 @@ def _HuberLogNormalDFE():
 
 
 _species.add_dfe(_HuberLogNormalDFE())
+
+
+def _KyriazisDFE():
+    id = "Mixed_K23"
+    description = "Deleterious Gamma DFE with additional lethals"
+    long_description = """
+    Return neutral and negative MutationType()s representing a human DFE.
+    Kyriazis et al. (2023), https://doi.org/10.1086/726736.
+    This model is similar to the Kim et al. (2017) DFE based on human
+    genetic data, and with dominance distribution from Henn et al. (2016).
```

Comment:
```suggestion
    genetic data, modified to include the dominance distribution from Henn et al. (2016).
```
is this what you mean?